selfdrive/ui/ui.cc

```diff
@@ -169,6 +169,9 @@ static void update_state(UIState *s) {
       scene.satelliteCount = data.getMeasurementReport().getNumMeas();
     }
   }
+  if (sm.updated("gpsLocationExternal")) {
+    scene.gpsAccuracy = sm["gpsLocationExternal"].getGpsLocationExternal().getAccuracy();
```

Comment:
not necessary anymore, you can just directly use submaster now

Reply:
I don't understand this comment, what can I remove?

Reply:
Oh, you mean from the other thread? Still, I like this abstraction